chart/compass/charts/default-tenant-mapping-handler/templates/rbac.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "defaultTenantMappingHandler.labels" . | nindent 4 }}
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ template "fullname" . }}
+  namespace: {{ $.Release.Namespace }}
+  labels:
+    app: {{ $.Chart.Name }}
+    release: {{ $.Release.Name }}
```

Comment:
everything below the `app` label should be covered with `{{- include "defaultTenantMappingHandler.labels" . | nindent 4 }}`